translations/ru-RU.ts

```diff
@@ -896,7 +896,7 @@
     <message>
       <location filename="../src/w_PrefrencesWindow.cpp" line="334"/>
       <source>tProxy is not supported on macOS and Windows</source>
-      <translation type="unfinished">tProxy is not supported on macOS and Windows</translation>
+      <translation>tProxy не поддерживается на MacOS и Windows</translation>
```

Comment:
@thebadgateway macOS should start in lowercase

Reply:
I apologize, did not know that they are different. I read Wikipedia.

Reply:
It's fine. I knew about that just a few days ago as well. We should thank @DuckSoft for #74

Reply:
xDD

Reply:
Sir, is the capital letter issue now fixed?
v1.3.7.2 will be out these days, and this PR will be merged after a few tests.

Reply:
I'll be at the computer in an hour and fix it right away

Reply:
Don't worry, It's midnight now in my country. I'll to that tomorrow or the day after.